src/image.c

```diff
@@ -959,6 +959,10 @@ int wolfBoot_verify_integrity(struct wolfBoot_image *img)
 {
     uint8_t *stored_sha;
     uint16_t stored_sha_len;
+#ifdef STAGE1_AUTH
+    /* Override global */
+    uint8_t digest[WOLFBOOT_SHA_DIGEST_SIZE];
```

Comment:
Isn't there already a global with this name? The comment makes it clear you want to use the local here. Can you think of a better way to do this?